README.md

```diff
@@ -321,9 +321,20 @@ The easiest way to achieve that is by using ActiveRecord's `ConnectionManagement
 `config.ru` before mounting Grape, e.g.:
 
 ```ruby
```

Comment:
I'd like to nitpick here. Let's make sections to keep things organized.

#### Rails 4

```ruby
use ...
```

#### Rails 5+

```ruby
...
```